app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new; end
```

Comment:
Are you using this action? If not, you should remove! It does not seem like you are [OPTIONAL]